prow/config/plugins.yaml

```diff
@@ -118,6 +118,15 @@ blockades:
       This PR modifies the files under the docs folder and requires the docs team to follow up on the PR.
 
       /label docs
+  - repos:
+      - pingcap/tidb
+    blockregexps:
+      - ^sessionctx/variable/
+      - ^config/config.toml.example$
+    explanation: |
+      This PR modifies significant config files which require review from domain experts.
+      
+      /cc qw4990 bb7133 XuHuaiyu cfzjywxk morgo
```

Comment:
@qw4990 @bb7133 @XuHuaiyu @cfzjywxk @morgo Please be aware of this change. If a PR touches config files, _all_ of you will receive a review request.